convokit/coordination/coordination.py

```diff
@@ -169,8 +179,18 @@ def summarize(self, corpus: Corpus, speakers: Collection[Union[Speaker, str]],
             raise Exception("Must fit before calling score")
 
         if split_by_attribs is None: split_by_attribs = []
-        if speaker_attribs is None: speaker_attribs = dict()
-        if target_attribs is None: target_attribs = dict()
+        if speaker_attribs is None:
+            speaker_attribs = dict()
+        else:
+            deprecation("Coordination's speaker_attribs parameter",
+                'speaker_utterance_selector')
+            raise ValueError
+        if target_attribs is None:
+            target_attribs = dict()
+        else:
+            deprecation("Coordination's target_attribs parameter",
+                'target_utterance_selector')
+            raise ValueError
```

Comment:
Will this break backwards compatibility?

Reply:
just added a few lines to reproduce the old behavior when these arguments are supplied